src/ocean/text/Arguments.d

```diff
@@ -1114,6 +1116,26 @@ public class Arguments
         return result;
     }
 
+    /// Ditto
```

Comment:
Not ditto, as the params are different.

Reply:
Then by definition you can never use `ditto` for the same overload set.
AFAIK ddoc supports that usage (and it's widely used in Phobos), but maybe we should document in which case we can use ditto then, since expectations seems to vary.

Reply:
Seems so. My expectation is that literally the same comment would be applicable to the `ditto`ed thing.

Reply:
Well actually most the the description was wrong as well, so Ditto was not applicable here.